projects/assets/src/translations/en/user.ts

```diff
@@ -51,7 +51,7 @@ export const user = {
       label: 'Confirm password',
       placeholder: 'Confirm Password',
     },
-    managmentInMyAccount: 'Management in My Account.',
+    managementInMyAccount: 'Management in My Account.',
```

Comment:
Translations breaking change (even though it's a typo fix). Created https://github.com/SAP/spartacus/issues/9038 to document it in the migration docs.

Reply:
TY